simd/src/Kokkos_SIMD_AVX2.hpp

```diff
@@ -814,8 +814,8 @@ class simd<std::uint64_t, simd_abi::avx2_fixed_size<4>> {
   template <class U, std::enable_if_t<std::is_convertible_v<U, value_type>,
                                       bool> = false>
   KOKKOS_IMPL_HOST_FORCEINLINE_FUNCTION simd(U&& value)
-      : m_value(_mm256_set1_epi64x(bit_cast<std::int64_t>(value_type(value)))) {
-  }
+      : m_value(_mm256_set1_epi64x(
+            Kokkos::bit_cast<std::int64_t>(value_type(value)))) {}
```

Comment:
Ping @ldh4 @ibaned this function has to be qualified to avoid ambiguities with C++20 if the `<bit>` header is transitively included.  Just FYI.